src/main/java/uk/gov/digital/ho/hocs/info/api/EntityResource.java

```diff
@@ -33,8 +33,10 @@ public ResponseEntity<GetCaseSummaryFieldsResponse> getCaseSummary(@PathVariable
     }
 
     @GetMapping(value = "/entity/list/{name}", produces = APPLICATION_JSON_UTF8_VALUE)
-    public ResponseEntity<List<EntityDto>> getEntitiesForListName(@PathVariable String name) {
-        List<Entity> entities = entityService.getByEntityListName(name);
+    public ResponseEntity<List<EntityDto>> getEntitiesForListName(@PathVariable String name,
```

Comment:
Thinking about this would this be beneficial returning the entire entity list all of the time with active flag? It then becomes a concern of the retrieved service whether to strip out inactive entities? 

This might be worth a development huddle on 🙂

Reply:
From my perspective it would make sense and be a lot simpler to return all of the entities every time. The vast majority of the time we do want every entity, it's only for active forms that we need to filter.

The only question for me is, how important is backwards compatibility here? as this change would stop the endpoint from being backwards compatible

Reply:
I think having a major bump is okay if it's for the right reasons. We'd have to make sure that it's correctly flagged as a major and manage it through to release.

Reply:
Agreed, I've made the change here so all values are returned and changed the PR label to major